### ISSUE TITLE
[17.0] endpoint_route_handler: Rule endpoint_route_handler edit has no group, this is a deprecated feature. Every access-granting rule should specify a group.

### ISSUE THREAD

**Issue by oerp-odoo** [OCA/web-api#62]
`17.0` deprecated access rights without specifying access group. In this module: endpoint_route_handler, there are two such groups:
`access_endpoint_route_handler_tool_edit` and `access_endpoint_route_handler_edit`. Now odoo is throwing warnings about that.

I wonder what default group it should get? Should this be publicly accessible?

**Comment by simahawk** [comment]
Yeah, that ACL is there only because we needed a tool model (which BTW I can now remove since I've found a better workaround).
